x-pack/packages/kbn-elastic-assistant/impl/assistant/quick_prompts/quick_prompt_selector/quick_prompt_selector.tsx

```diff
@@ -138,14 +138,24 @@ export const QuickPromptSelector: React.FC<Props> = React.memo(
       return (
         <EuiFlexGroup
           alignItems="center"
-          justifyContent="spaceBetween"
           className={'parentFlexGroup'}
+          component={'span'}
+          justifyContent="spaceBetween"
         >
-          <EuiFlexItem grow={false}>
-            <EuiHealth color={color}>
-              <span className={contentClassName}>
-                <EuiHighlight search={searchValue}>{label}</EuiHighlight>
-              </span>
+          <EuiFlexItem
+            component={'span'}
+            grow={false}
+            css={css`
+              width: calc(100% - 60px);
+            `}
+          >
+            <EuiHealth
+              color={color}
+              css={css`
+                overflow: hidden;
```

Comment:
there's no `...` truncation on the quick prompt titles because the `EuiHealth` has flex group descendants that can't be styled via the public EUI api